framework/skill-levels.json

```
@@ -1,32 +1,42 @@
 [
     {
-        "Level": 0,
-        "Name": "None or N/A",
-        "Description": "You do not possess this competency or are not required to apply or demonstrate this competency (this competency is not applicable to your position)."
+        "skill_level": "Level 0",
+        "level": 0,
+        "short_name": "None or N/A",
+        "short_description": "",
+        "long_description": "No skill or ability demonstrated, or the skill is not required or applicable to the role.",
+        "focus": ""
     },
     {
-        "Level": 1,
-        "Name": "Fundamental awareness (basic knowledge)",
-        "Description": "You have a common knowledge or an understanding of basic techniques and concepts. Focus is on learning and remembering."
+        "skill_level": "Level 1",
+        "level": 1,
+        "short_name": "Awareness",
+        "short_description": "Fundamental awareness (basic knowledge) or fundamental ability",
+        "long_description": "Technical: has basic knowledge of the area and understands core techniques and concepts;\ncan follow instructions and perform simple tasks with support, but the application of the skill is limited.\nNon-technical: recognises the importance of the skill, shows initial effort, and applies it inconsistently or only in simpler contexts.\n",
```

```suggestion
        "long_description": "Technical: has basic knowledge of the area and understands core techniques and concepts;\ncan follow instructions and perform simple tasks with support, but the application of the skill is limited.\nNon-technical: recognises the importance of the skill, shows initial effort, and applies it inconsistently or only in simpler contexts.",
```
Do these trailing line breaks matter? I will propose taking them all out here.